homeassistant/components/frontend/__init__.py

```diff
@@ -169,6 +173,15 @@ def register_panel(hass, component_name, path, md5=None, sidebar_title=None,
             'get', '/{}/{{extra:.+}}'.format(url_path), index_view.get)
 
 
+@bind_hass
+def add_extra_html_urls(hass, urls):
```

Comment:
Let's just take in 1 url.

I'm actually not too convinced that we need this method just yet. What would be a usecase that we want to programmatically add these?

Reply:
Changed to take 1 url